Access Medicine.js

```diff
@@ -0,0 +1,264 @@
+{
+	"translatorID": "60e55b65-08cb-4a8f-8a61-c36338ec8754",
+	"label": "Access Medicine",
+	"creator": "Jaret M. Karnuta",
+	"target": "^https?://(0-)?accessmedicine\\.mhmedical\\.com",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2016-12-26 19:52:12"
+}
+
+function detectWeb(doc, url){
+	var citation = doc.getElementById("hfGetCitation");
+	if(citation){
+		return "bookSection";
+	}
+}
```

Comment:
Added for search results, but parsing whole books (e.g., http://accessmedicine.mhmedical.com/book.aspx?bookID=1130) isn't possible as the content I need is loaded over ajax when the user clicks a node (Let me know if there is a workaround for this)